src/style/properties.js

```diff
@@ -425,9 +425,9 @@ type PossiblyEvaluatedValue<T> =
 export class PossiblyEvaluatedPropertyValue<T> {
     property: DataDrivenProperty<T>;
     value: PossiblyEvaluatedValue<T>;
-    globals: GlobalProperties;
+    globals: EvaluationParameters;
```

Comment:
I'm not very familiar with this part of the code, but does it make sense to rename `globals` now that the type isn't `GlobalProperties` before?